src/middlewared/middlewared/scripts/configure_fips.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+import os
+import re
+import sqlite3
+import subprocess
+
+from middlewared.utils.db import query_config_table
+
+
+FIPS_MODULE_FILE = '/usr/lib/ssl/fipsmodule.cnf'
+OPENSSL_CONFIG_FILE = '/etc/ssl/openssl.cnf'
+OPENSSL_FIPS_FILE = '/etc/ssl/openssl_fips.cnf'
+RE_INCLUDE_FIPS = re.compile(fr'\s*\.include {re.escape(OPENSSL_FIPS_FILE)}\s*$')
+
+
+def validate_system_state() -> None:
+    for path in (FIPS_MODULE_FILE, OPENSSL_CONFIG_FILE, OPENSSL_FIPS_FILE):
+        if not os.path.exists(path):
+            raise Exception(f'{path!r} does not exist')
+
+
+def modify_openssl_config(enable_fips: bool) -> None:
+    with open(OPENSSL_CONFIG_FILE, 'r') as f:
```

Comment:
Does anybody else modify `OPENSSL_CONFIG_FILE`? Why do we have to edit it instead of generating from scratch every time like we do with other config files?

Reply:
Will make the changes consuming it from `/conf/base` itself as discussed. Thanks!